### ISSUE TITLE
Mismatched javadoc in Zoomer.java

### ISSUE THREAD

**Issue by mikhalkovnik** [GSI-CS-CO/chart-fx#212]
Hi, I just noticed some mismatching in the javadoc:
[Zoomer#setPannerEnabled](https://github.com/GSI-CS-CO/chart-fx/blob/master/chartfx-chart/src/main/java/de/gsi/chart/plugins/Zoomer.java#L574): javadoc `@link` to[ `#sliderVisibleProperty()` ](https://github.com/GSI-CS-CO/chart-fx/blob/master/chartfx-chart/src/main/java/de/gsi/chart/plugins/Zoomer.java#L668) instead of [`#pannerEnabledProperty()`](https://github.com/GSI-CS-CO/chart-fx/blob/master/chartfx-chart/src/main/java/de/gsi/chart/plugins/Zoomer.java#L527)

Affected Versions: up to [Latest](https://github.com/GSI-CS-CO/chart-fx/releases) ([11.1.5](https://github.com/GSI-CS-CO/chart-fx/releases/tag/11.1.5) and [8.1.5](https://github.com/GSI-CS-CO/chart-fx/releases/tag/8.1.5) at issue opening)

**Comment by RalphSteinhagen** [comment]
@mikhalkovnik well spotted. Fixed and will be merged into master once the dev-x.2.0 cycle is finished.
